wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -39,78 +59,22 @@
 							<div class="tile">
 								<h4>Embeddable</h4>
 								<i class="fas fa-code"></i>
-								<p>Easily add the quiz to any page on your website with our simple embed code.</p>
+								<p>Easily add the quiz or buttons to any page on your website with our simple embed code.</p>
 							</div>
 							<div class="tile">
 								<h4>Gain insight</h4>
 								<i class="fas fa-chart-pie"></i>
-								<p>Test your audience’s knowledge and response rate. Even create A/B tests to see which quizzes are more effective.</p>
+								<p>Test your audience’s knowledge and response rate. Easily gain back-end access to the data from our tools!.</p>
 							</div>
 						</div>
 					</div>
 				</section>
-
-				<section class="callout quiz-creator-research">
-					<div class="container container--xl wrapper--pad">
-
-							<h4 class="subhead">A quiz tool designed to increase civic engagement</h4>
-							<h2 class="headline">Challenge your Audience</h2>
-							Our research shows that quizzes help people learn more than presenting information without a quiz. Even more, they increase time on page and site visitors rate them as enjoyable.
-
-							<a href="/research/online-polls-and-quizzes/">Explore White Paper</a>
 					</div>
 				</section>
```

Comment:
There are three `</div></section>` tags in a row at the bottom of this file. I think most of them can be removed.